### ISSUE TITLE
You already have looked for webTorrent?

### ISSUE THREAD

**Issue by matheuschimelli** [PopcornRepos/PopcornTimeOnline#19]
You already have looked for webTorrent popcorn time implementation?
[https://github.com/feross/webtorrent](WebTorrent Github)

**Comment by omgbox** [comment]
webtorrent at this time is not compatible with the real torrent network. webTorrent uses its own webRTC peer to peer system which is based on Bit-torrent but is not compatible. 

in firefox there is a native torrent download plug-in called https://addons.mozilla.org/en-GB/firefox/addon/torrent-tornado/

somebody needs to go and look how his able to download torrents and cross link that plug-in instead of peerflix.

**Comment by matheuschimelli** [comment]
I think that download a program to use popcor time cant be safe, because i
dont know the torrent time source, and this not is open source.
Em 12/02/2016 10:39 PM, "lib-noob" notifications@github.com escreveu:

**Comment by DevLight1** [comment]
As we all found out torrent time has flaws that need to be fixed but i found this and it got me interested http://cinematrix.one/

**Comment by syl3n7** [comment]
Yeah, in that site, they use "https://webseed.btorrent.xyz" as a host for the videos
if you go to that link, it's all there

**Comment by matheuschimelli** [comment]
I think that is necessary create a webtorrent implementation

**Comment by DevLight1** [comment]
Already trying but still new to figure a solution for webtorrent

**Comment by EliteGams** [comment]
Hey, maybe we can create a working nodejs site with bittorrent protocol to allow the webtorrent to work with dth etc....  ??? Anyone think this might work ?

**Comment by albabosh** [comment]
That is one https://github.com/KeizerDev/Browsertime
